NEWS.md

```diff
@@ -84,6 +84,7 @@ This is a major release which rewrites the core code to be asynchronous.
 - **toggle**: `text` option is removed and now you can use `format` to set the text
 - **notmuch**: `name` option is removed and now you can use `format` to set the name
 - **{apt, dnf, pacman}**: `hide_when_uptodate` option is removed and now you can use `format_up_to_date = ""` to hide the block
+- **temperature**: `collapsed` option is removed and now you can use `format_alt = ""` to maintain the behavior
```

Comment:
`" $icon "` would be more correct. `""` will hide the block forever and the user will have nowhere to click to show it again.